alan_std.js

```diff
@@ -1058,7 +1058,9 @@ export function frameFramebuffer(frame) {
 
 export async function runWindow(initialContextFn, contextFn, gpgpuShaderFn) {
   // None of this can run before `document.body` exists, so let's wait for that
-  await new Promise((r) => document.addEventListener("DOMContentLoaded", () => r()));
+  if (document.readyState !== "complete" && document.readyState !== "loaded") {
+    await new Promise((r) => document.addEventListener("DOMContentLoaded", () => r()));
+  }
```

Comment:
If this window is started *after* the initial loading of the page, this `await` would cause it to hang forever before, so now check if we actually need the `await`.